### DISCUSSION TITLE
YOLOv5 ver 6 model included BiFPN?

### DISCUSSION THREAD

**Discussion by ys31jp** [ultralytics/yolov5#5328] [Q&A] [answered]
I notice there is in the models/hub yolov5-bifpn.yaml. is the latest yolov5 model using BiFPN then PANet?

**Comment by glenn-jocher** [comment, marked as answer]
@ys31jp yes, there is a YOLOv5-BiFPN model in hub. This is a v5.0 model. We trained this model to match the BiFPN head of EfficientDet, but did not see substantial mAP gains with this change, and we learned that the current YOLOv5 PANet head is actually very similar to what the EffiicentDet authors call a BiFPN head.
https://github.com/ultralytics/yolov5/blob/master/models/hub/yolov5-bifpn.yaml

<img width="1196" alt="Screen Shot 2021-10-25 at 3 34 05 PM" src="https://user-images.githubusercontent.com/26833433/138705293-d21509bb-758a-49ac-96fa-0f354247ef54.png">

**Comment by ys31jp** [comment]
Thanks for replied. Great to know this. Will check on the similarity for the BiFPN on the model.
